packages/common-scripts/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # @ckb-lumos/common-scripts
 
+## 0.22.0
+
+### Minor Changes
+
+- [#596](https://github.com/ckb-js/lumos/pull/596) [`ad483d9`](https://github.com/ckb-js/lumos/commit/ad483d9cb55ad38db9604fde456033b9513bc7f1) Thanks [@homura](https://github.com/homura)! - feat: support registered lock in dao operations
+
+- [#607](https://github.com/ckb-js/lumos/pull/607) [`b44fb39`](https://github.com/ckb-js/lumos/commit/b44fb3967d8c0cec40e8a39a885b33a515c2d9d4) Thanks [@homura](https://github.com/homura)! - feat: supported bitcoin auth in omnilock
+
+- [#623](https://github.com/ckb-js/lumos/pull/623) [`028d28e`](https://github.com/ckb-js/lumos/commit/028d28e463fd2f8b58b722684ad9fac982b3b923) Thanks [@homura](https://github.com/homura)! - feat: omnilock bitcoin auth message is changed to `CKB (Bitcoin Layer) transaction: 0x`
```

Comment:
This is a breaking change

Reply:
this is a new feature without breaking change by adding an auth ID 0x04 for Bitcoin

Reply:
Got it at https://github.com/ckb-js/lumos/pull/623/files#r1461360809